gradio_interface.py

```
@@ -293,7 +293,7 @@ def reset_to_defaults():
 def initialize_engine(
     model_path: str = DEFAULT_MODEL_PATH,
     audio_tokenizer_path: str = DEFAULT_AUDIO_TOKENIZER_PATH,
-    tokenizer_path: str = None,
+    tokenizer_path: str = DEFAULT_MODEL_PATH,
     device: str = None,
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)